stdlib/asm/crypto/stark/constants.masm

```diff
@@ -234,6 +241,18 @@ export.domain_offset_inv
     push.DOMAIN_OFFSET_INV
 end
 
+export.lde_domain_generator_ptr
+    push.LDE_DOMAIN_GEN_PTR
+end
```

Comment:
Should this be `get_domain_generator`?

(relevant for other procedures in this module as well)